script/authors

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+
+# Creates a list of all authors in the alphabetical order respecting .mailmap.
```

Comment:
What is `.mailmap.`?

Reply:
@zeke 
Good question ) Here's a description: https://git-scm.com/docs/git-check-mailmap#_mapping_authors
Also added a comment in the script with the same link.